content/blogs/2024-02-06-gitops.md

```diff
@@ -0,0 +1,116 @@
+
+- [Git Push task](https://kestra.io/plugins/plugin-git/tasks/io.kestra.plugin.git.push): once you’ve developed your flow in the UI you want to push it to your Git repository. The Git Push task allows you to push any flow or files stored in the Namespace File editor to a dedicated Git branch. This way you can develop in Kestra UI and save the changes in Git.
+
+- [Git Sync task](https://kestra.io/plugins/plugin-git/tasks/io.kestra.plugin.git.sync) in Kestra ensures your development stays in sync with Git. Making Git the single source of truth. If you prefer developing in the UI but want Git to hold all changes, use the Git Sync task to align your Kestra instance with Git. This maintains a clean production environment by overwriting any UI changes.
+
+![version-control-with-git](/blogs/2024-02-06-gitops/version-control-with-git.png)
+
+**Using IDE/Terminal (“As Code”):**
+
+You’re probably using an IDE if you navigate through different projects daily. [Kestra offers a VS Code extension](https://marketplace.visualstudio.com/items?itemName=kestra-io.kestra) allowing full auto-completion on Flow YAML declaration, making development in IDE very productive.
+Once our code is in Git we want to deploy it to the final production environment. This is usually done through CI/CD.
+
+Kestra supports many CI/CD engines like GitHub Actions, GitLab CI/CD Pipeline, Azure DevOps Pipeline, and BitBucket pipes. It also offers a CLI to integrate into any system.
+Moreover, users already using Terraform will love to declare flows, secrets, and any Kestra asset as Terraform resources so they can be easily deployable across environments.
+
+![dev to prod](/blogs/2024-02-06-gitops/dev-to-prod.png)
+
+## Need For Git: No Drift
+
+Nowadays version control with Git and Infrastructure as Code are non-negotiable. They are needed to scale and collaborate effectively.
+
+At Kestra we firmly believe that the GitOps paradigm will extend far beyond its current prominence in software. Data already initiates this shift, but Sales, CRM, HR, Revenue, and every part of the company will find themselves in the need for scalability, collaboration management, and robust safeguarding.
+
+Implementing GitOps enables a holistic view of the company stack. It gathers everyone at the same table.
+Having a robust, fast, and consistent platform overcomes the challenges posed by hyperspecialization in various domains — be it data orchestration, process, and microservice orchestration, or infrastructure and business automation.
+
+Applying GitOps aims to provide support for projects to be consistent, manageable, and transparent. That’s what Kestra provides too.
+
+Are you already implementing GitOps in your projects? What are the benefits?
+Please share your views and come to [Kestra’s Slack](https://kestra.io/slack) channel.
+
+Also, go through [Kestra's GitHub repository and add a star!](https://github.com/kestra-io/kestra) Follow us on [Twitter](https://twitter.com/kestra_io)
```

Comment:
```suggestion
At Kestra, we believe that the GitOps paradigm will extend far beyond its current prominence in software. Data already initiates this shift, but Sales, CRM, HR, Revenue, and every part of the company will find themselves in the need for scalability and collaboration.

Implementing GitOps enables a holistic view of the company stack. Having a robust, fast, and consistent platform overcomes the challenges posed by hyperspecialization in various domains — be it data orchestration, process and microservice orchestration, or infrastructure and business automation.

Applying GitOps aims to provide support for projects to be consistent, manageable, and transparent. That’s what Kestra provides, too.

Are you already implementing GitOps in your projects? What are the benefits?
Please share your views in the [Kestra’s Slack](https://kestra.io/slack) channel.

If you want to learn more about Kestra, check out our [documentation](https://kestra.io/docs) or [request a demo](https://kestra.io/demo), and if you like the project, become our next star on [GitHub](https://github.com/kestra-io/kestra).
```

what happened to that last sentence haha

Reply:
done ;) (didn't know what happened, too fast copy-past 😅)